README.md

```diff
@@ -1,220 +1,130 @@
 # c-lightning: A specification compliant Lightning Network implementation in C
 
-c-lightning is a [standard compliant][std] implementation of the Lightning
-Network protocol.
-The Lightning Network is a scalability solution for Bitcoin, enabling
-secure and instant transfer of funds between any two parties for any
-amount.
+c-lightning is a lighweight, highly customizable and [standard compliant][std] implementation of the Lightning Network protocol.
 
-[std]: https://github.com/lightningnetwork/lightning-rfc
-
-For more information about the Lightning Network please refer to
-http://lightning.network.
 
 ## Project Status
 
 [![Build Status][travis-ci]][travis-ci-link]
 [![Pull Requests Welcome][prs]][prs-link]
 [![Irc][IRC]][IRC-link]
-[![Documentation Status](https://readthedocs.org/projects/lightning/badge/?version=docs)](https://lightning.readthedocs.io/)
-
-[travis-ci]: https://travis-ci.org/ElementsProject/lightning.svg?branch=master
-[travis-ci-link]: https://travis-ci.org/ElementsProject/lightning
-[prs]: https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat
-[prs-link]: http://makeapullrequest.com
-[IRC]: https://img.shields.io/badge/chat-on%20freenode-brightgreen.svg
-[IRC-link]: https://webchat.freenode.net/?channels=c-lightning
+[![Documentation Status](https://readthedocs.org/projects/lightning/badge/?version=docs)][docs]
 
-This implementation is still very much a work in progress.
-It can be used for testing, but __it should not be used for real funds__.
-We do our best to identify and fix problems, and implement missing
-features.
+This implementation has been in production use on the Bitcoin mainnet since early 2018, with the launch of the [Blockstream Store][blockstream-store-blog].
+We recommend getting started by experimenting on `testnet`, but the implementation is considered stable and can be safely used on mainnet.
```

Comment:
I guess the testnet link is only helpful for developers and not for endusers. When I was new to the project it took me almost 2 months to figure out there is regtest so we might want to add a link to this file https://github.com/shesek/spark-wallet/blob/master/doc/dev-regtest-env.md or at least copy those instructions to our doc respository.

Reply:
Yes, regtest is usually what you want for testing anyway.  There's also contrib/startup_regtest.sh.